### PR TITLE
removed eslint warnings from chain and cli module

### PR DESCRIPTION
**Pull request by sadiq1971**

### CODE REVIEW

**Comment by GregTheGreek** [comment]
hi @sadiq1971 thanks for the PR!

Looks like you have a bit of a merge conflict, could you check that out :)

**Comment by sadiq1971** [comment]
> hi @sadiq1971 thanks for the PR!
> 
> Looks like you have a bit of a merge conflict, could you check that out :)

Hi @GregTheGreek , I have resolved the conflict. Take a look.

**Review by wemeetagain — CHANGES_REQUESTED** [review]
You can remove the unused import statements
Other than that looks good!

File: src/chain/index.ts [hunk 1]
```diff
@@ -60,7 +62,8 @@ export class BeaconChain extends EventEmitter {
   /**
    * Initialize the beacon chain with a genesis beacon state / block
    */
-  public async initializeChain(genesisTime: number64, genesisDeposits: Deposit[], genesisEth1Data: Eth1Data): Promise<void> {
+  public async initializeChain(genesisTime: number64, genesisDeposits: Deposit[]
```
Comment (GregTheGreek) [inline src/chain/index.ts]: I'd prefer if everything was on its own line for constructors
```suggestion
  public async initializeChain(
    genesisTime: number64, 
    genesisDeposits: Deposit[],
```

File: src/chain/index.ts [hunk 2]
```diff
@@ -60,7 +62,8 @@ export class BeaconChain extends EventEmitter {
...
+    , genesisEth1Data: Eth1Data): Promise<void> {
```
Comment (GregTheGreek) [inline src/chain/index.ts]:
```suggestion
    genesisEth1Data: Eth1Data): Promise<void> {
```

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

**Comment by sadiq1971** [comment]
It seems some new file have been added to the master branch in the previous PR . So after resolving conflict i am having error in my local branch. So I am closing this PR and Opening a Updated PR

**Pull request closed without merging**